homeassistant/components/backup/util.py

```diff
@@ -117,6 +118,14 @@ def read_backup(backup_path: Path) -> AgentBackup:
         )
 
 
+def suggested_filename(backup: AgentBackup) -> str:
+    """Suggest a filename for the backup."""
+    date = dt_util.parse_datetime(backup.date, raise_on_error=True)
+    return "_".join(
+        f"{backup.name} - {date.strftime('%Y-%m-%d %H.%M %S%f')}.tar".split()
+    )
```

Comment:
Can we use `replace(" ", "_")` instead of `split` -> `join`, since it would make this a bit clearer? Or is there any downside?

Reply:
No, because that only replaces single spaces with single underscores.
This solution replaces any combination of white space with a single underscore.